docs/kubernetes/operator/README.md

```diff
@@ -85,15 +87,19 @@ The Helm Chart is configured to enable zero-code instrumentation using the [Oper
 
 ## Deploying components using Kibana Onboarding UX
 
-The preferred method for deploying all components is through the Kibana Onboarding UX. Follow these steps:
+The preferred method for deploying all components is through Kibana Onboarding UX. Follow these steps:
 
 1. Navigate in Kibana to **Observability** --> **Add data**
```

Comment:
good point. I didn't want to be super clear or specific on the navigation path because the `Add data` appears almost everywhere :-D.  Also when you initially open Kibana the first time in a deployment.

I don't know how exactly the `Add data` appears in serverless, I was assuming it would be intuitive, and the idea was to be platform agnostic in this doc.

Sadly the `search bar` cannot be used for `Add data`.